bin.src/run_generic.py

```diff
@@ -92,6 +95,8 @@ def parseArgs(subdir='out', parser=None):
                              " e.g.: 'night <= 365' or 'propId = 5' "
                              " (**may not work with slew batches)")
     parser.add_argument("--nyears", type=int, default=10, help="Number of years in the run (default 10).")
+    parser.add_argument("--ditherStacker", type=str, default=None, help="Name of dither stacker to use"
```

Comment:
Can we add an example of a common dither stacker that would be used here? Something similar to how an example is given for the `--sqlConstraint`.